File: curso_c_cpp/c/05_condicionales/ejemplos/par_impar.c
/*
 * if/else: decir si un numero es par o impar.
 */
#include <stdio.h>

int main(void) {
    int n;
    printf("Numero: ");
    scanf("%d", &n);

    if (n % 2 == 0) {
        printf("%d es par\n", n);
    } else {
        printf("%d es impar\n", n);
    }
    return 0;
}

File: curso_java/05_condicionales/ejemplos/ParImpar.java
/*
 * if/else: decir si un numero es par o impar.
 */
import java.util.Scanner;

public class ParImpar {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Numero: ");
        int n = sc.nextInt();

        if (n % 2 == 0) {
            System.out.println(n + " es par");
        } else {
            System.out.println(n + " es impar");
        }
        sc.close();
    }
}
